### ISSUE TITLE
NetRocks: mtime doesn't show up via SFTP

### ISSUE THREAD

**Issue by dayfuaim** [elfmz/far2l#2283]
При подключении через NetRocks, не показывается время изменения файлов/папок.
![Снимок экрана 2024-07-12 в 13 49 37](https://github.com/user-attachments/assets/7d5a7dce-e814-4b12-8081-8ddc86a8a372)

Протокол SFTP.

Пожалуйста, почините.

**Comment by akruphi** [comment]
У меня в последней версии из мастера (`490b3728-beta`) для NetRocks соединений SFTP время показывается как на панелях, так и по Ctrl+A (локальный far2l запущен на Ubuntu, сервера SFTP - Debian и FreeBSD). Какую версию far2l Вы используете (смотреть можно в заголовке окна или по `far:about`)? Откуда брали сборку?

**Comment by dayfuaim** [comment]
У меня macOS 10.15.7.
А вот far2l-ов оказалось два: 
один из homebrew:
![Снимок экрана 2024-07-12 в 22 26 38](https://github.com/user-attachments/assets/35a1ef3f-3e9f-48fc-b0a7-07ca5d623f02)
, а другой – какой-то старый:
![Снимок экрана 2024-07-12 в 21 35 43](https://github.com/user-attachments/assets/b594f2a4-b7a4-4752-80c2-82dcd4c618fe)

Но новый всё равно не показывает mtime:
![Снимок экрана 2024-07-12 в 22 22 47](https://github.com/user-attachments/assets/4e62a45a-2f15-4c59-b672-7e1a949a3f74)

По SFTP я подключаюсь к МакБуку. Обратная ситуация такая же: МакБук к Мак Мини не показывает mtime.

**Comment by akruphi** [comment]
У вас обе очень старые версии. Сейчас **2.6.1**. Попробуйте хотя бы последний релиз: https://github.com/elfmz/far2l?tab=readme-ov-file#osxmacos-binaries

**Comment by dayfuaim** [comment]
Взял последний релиз (v2.6.1 BETA + HOTFIX1), снёс все остальные.

Результат такой же. ¯\\_(ツ)_/¯

Видимо, Мак к Маку дают такое.

**Comment by akruphi** [comment]
> Видимо, Мак к Маку дают такое по SFTP.

Однако! Если есть возможность, пробуйте:
* другой альтернативный клиент (не far2l) - видно ли в нём дату файлов на стороне вашего SFTP или нет?
* скачать файл с той стороны и посмотреть какая дата получится у скаченного файла
    * при скачивании через far2l 
    * и при скачивании через альтернативный клиент?

**Comment by dayfuaim** [comment]
> 
> * другой альтернативный клиент (не far2l) - видно ли в нём дату файлов на стороне вашего SFTP или нет?

Попробовал mc – да, дату показывает.

> * скачать файл с той стороны и посмотреть какая дата получится у скаченного файла
>   
>   * при скачивании через far2l

Да, дата файла "с той стороны".

>   * и при скачивании через альтернативный клиент?

Да, дата тоже нормальная.

> * соединится с этим SFTP клиентом с другого компьютера (чем-то вообще не из мака)

С планшета – всё в порядке. И при копировании тоже.

**Comment by spnethw** [comment]
@dayfuaim  Cудя по заголовку 3 столбца ("Изменение"), вы используете свою, кастомную конфигурацию Режима панели?

@akruphi У меня в столбце "Изменение" тоже ничего не отображается, если его настроить в ```Параметры``` -- ```Режимы панели файлов```, но отображаются значения в столбцах "Запись" (DM) и "Доступ" (DA).

Полагаю, стобец "Изменение" это время изменения статуса, которое, как я понял, в принципе [не передается](https://github.com/elfmz/far2l/pull/2133)? Ну а "Время создания" в принципе не Linux-way, поэтому пустое.

**Comment by akruphi** [comment]
> У меня в столбце "Изменение" (DE) тоже ничего не отображается

Точно :fire: У меня в столбцах **Запись** и **Доступ** все даты показываются. Даже не обратил внимание, что тут нестандартный столбец. А вот в столбце **Создание** у меня даты показываются в SCP, SHELL и FTP, но пустота в SFTP.

**Comment by dayfuaim** [comment]
@spnethw Да, выставил "DМM" вместо "DEM" – даты отображаются и соответствуют. 👍 

Сортировка тоже работает.

Думаю, _данный_ issue можно закрывать.
